src/cli.yaml

```diff
@@ -0,0 +1,395 @@
+name: git-dit
+version: 0.1.0
+authors:
+    - Julian Ganz <neither@nut.email>
+    - Matthias Beyer <mail@beyermatthias>
+about: Distributed issue tracker for git
+args:
+    - verbose:
+        short: v
+        long: verbose
+        multiple: false
+        help: Print what you are doing
```

Comment:
This doesn't seem right. Verbose output doesn't just contain what the program does. "Verbose output" or something similarly vague would be fine.